modules/bookCollection.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/prefer-default-export */
+import displayBook from './bookDisplay.js';
+
+const bookContainer = document.querySelector('.bookDisplay');
+export class BookCollection {
+  constructor(bookArray) {
+    this.bookArray = bookArray;
+  }
+
+  removeBook(book) {
+    this.bookArray.splice(book, 1);
+    localStorage.setItem('bookCollectionArray', JSON.stringify(this.bookArray));
+    bookContainer.innerHTML = '';
+    this.bookArray.forEach((book, i) => {
+      book.id = i;
+      displayBook(book);
+      if (document.querySelector(`#remove-${book.id}`)) {
+        document
+          .querySelector(`#remove-${book.id}`)
+          .addEventListener('click', this.removeBook.bind(this, book));
+      }
+    });
+
+    if (this.bookArray[0]) bookContainer.style.border = 'solid 2.5px #635e5e';
+    else bookContainer.style.border = 'none';
+  }
```

Comment:
- [ ] when adding a duplicate book and removing the second instance the application deletes the first in the array instead of the second, you can solve this by using the ID of the book when removing it.